lang/rustup/Makefile

```diff
@@ -0,0 +1,70 @@
+#
+# Copyright (C) 2020 OpenWrt.org
```

Comment:
You can't assign copyright to openwrt.org, you either need to drop it, or copyright yourself.  (Unless you actually have a copyright assignment on file with ... SPI or SFC or whoevers' in charge of ti these days)

Reply:
Dropped.